src/Prometheus/Storage/APCng.php

```diff
@@ -438,11 +438,10 @@ private function buildPermutationTree(array $labelValues): \Generator /** @phpst
         if (count($labelValues) > 0) {
             $lastIndex = array_key_last($labelValues);
             $currentValue = array_pop($labelValues);
-            if ($currentValue != null) {
```

Comment:
This change seems safe to me, as `count() > 0` ensures that `array_pop()` won't return null (assuming that the array element itself won't evaluate to `null`)